nncf/experimental/post_training/algorithms/fast_bias_correction/algorithm.py

```diff
@@ -0,0 +1,312 @@
+from nncf.common.graph.transformations.commands import TargetPoint
+from nncf.experimental.post_training.algorithms import AlgorithmParameters
+from nncf.experimental.post_training.algorithms.algorithm import Algorithm
+from nncf.experimental.post_training.algorithms.algorithm import PostTrainingAlgorithms
+from nncf.experimental.post_training.algorithms.fast_bias_correction.backend import ALGO_BACKENDS
+from nncf.experimental.post_training.api.engine import Engine
+from nncf.experimental.post_training.factories import NNCFGraphFactory
+from nncf.experimental.post_training.graph.model_transformer import StaticModelTransformerBase
+from nncf.experimental.post_training.statistics.statistic_point import StatisticPoint
+from nncf.experimental.post_training.statistics.statistic_point import StatisticPointsContainer
+
+ModelType = TypeVar('ModelType')
+
+
+class FastBiasCorrectionParameters(AlgorithmParameters):
+    """
+    Base class of FastBiasCorrection parameters.
+    """
+
+    def __init__(self, number_samples: int = 100, threshold: float = 2.0) -> None:
+        """
+        Initial method for the FastBiasCorrectionParameters
+
+        :param number_samples: number of the samples for the statistics collection
+        :param threshold: threshold that regulates the application of the shift (or not)
+        """
+        self.number_samples = number_samples
+        self.threshold = threshold
+
+    def to_json(self) -> Dict[str, Union[str, float, int]]:
+        """
+        Serialize all FastBiasCorrection parameters to JSON.
+        """
+
+
+class FastBiasCorrection(Algorithm):
+
+    def __init__(self, parameters: FastBiasCorrectionParameters):
+        super().__init__()
+        self.number_samples = parameters.number_samples
+        self.threshold = parameters.threshold
+        self.nncf_graph = None
+        self._backend_entity = None
+
+    @property
+    def available_backends(self) -> List[BackendType]:
+        return ALGO_BACKENDS.registry_dict
+
+    def _set_backend_entity(self, model: ModelType) -> None:
+        """
+        Factory method for creating the needed backend based on the model
+
+        :param model: backend-specific input model
+        """
+        model_backend = get_backend(model)
+        if model_backend == BackendType.ONNX:
+            from nncf.experimental.post_training.algorithms.fast_bias_correction.onnx_backend import ONNXFBCAlgoBackend
+            self._backend_entity = ONNXFBCAlgoBackend()
+        else:
+            raise RuntimeError('Cannot return backend-specific entity'
+                               'because {} is not supported!'.format(model_backend))
+
+    def _apply(self, model: ModelType, engine: Engine,
+               statistic_points: StatisticPointsContainer) -> ModelType:
+        self._set_backend_entity(model)
+
+        model_transformer = self._backend_entity.model_transformer(model)
+        transformation_layout = TransformationLayout()
```

Comment:
It does not look good to me that we should create some layout objects explicitly in the algorithm. Why just not to:
1. Create a command
2. Put it in the ModelTransformer
3. Call .transform() when there is a scope of commands at some point

Reply:
Let's discuss it offline.